src/client/src/apps/MainRoute/widgets/spotTile/selectors.ts

```diff
@@ -23,37 +23,51 @@ const DEFAULT_TILE_DATA: SpotTileData = {
   rfqState: 'none',
   rfqPrice: null,
   rfqReceivedTime: null,
-  rfqTimeout: null
+  rfqTimeout: null,
 }
 
-const getCurrencyPair = (state: GlobalState, currencyPairId: string): CurrencyPair =>
-  state.currencyPairs[currencyPairId]
+const getCurrencyPair = (
+  state: GlobalState,
+  currencyPairId: string | undefined,
```

Comment:
most of this file changes are formatting. Here we see actual code changes - adding handling of `undefined`